doc/source/whatsnew/v0.21.0.txt

```diff
@@ -566,7 +566,9 @@ Categorical
 - Bug in the categorical constructor with empty values and categories causing
   the ``.categories`` to be an empty ``Float64Index`` rather than an empty
   ``Index`` with object dtype (:issue:`17248`)
-
+- Bug in categorical operations on Series like `Series.cat.set_categories`
```

Comment:
Sorry, it should be double backticks, not single

    ``Series.cat.set_categories``